spec/cypress/integration/admin-create-administrator.js

```diff
@@ -20,7 +20,7 @@ describe("Admin user creating another admin user", () => {
     cy.get("input[name='user[email]']").type(email);
     cy.get(".govuk-button").contains("Continue").click();
 
-    cy.get("h1").should("equal", "Confirm these details");
+    cy.get("h1").should("contain", "Confirm these details");
```

Comment:
For future reference, it's "eql", not "equal" (I have no idea why)

Reply:
Callum - could you create a dev story for the next sprint for exploring page object pattern in cypress

Reply:
Can you use expectations syntax in cypress